### PR TITLE
[#831] Added the invoke command for legion cli

### LINKED ISSUE #831: Add "invoke" legionctl command
As a legion user, I want a legionctl command that allows me to invoke the deployed model.

### PR DESCRIPTION
**Pull request by mcdoker18**
Added command for locally or remotely(on cluster) invocation of models.
Example of usages locate in the docs and robot tests.

This closes #831

### CODE REVIEW

**Comment by mcdoker18** [comment]
bot, test all

File: legion/bin/legionctl [hunk 1]
```diff
@@ -30,6 +30,8 @@ if __name__ == '__main__':
     legion.cli.configure_logging(args)
     root_logger = logging.getLogger()
 
+    root_logger.debug('CLI parameters: %s', args)
```
Comment (kirillmakhonin) [inline legion/bin/legionctl:33]: We're not using % formatting, even for logs. We're using `format`

File: legion/legion/cli.py [hunk 2]
```diff
@@ -678,8 +755,8 @@ def build_parser():  # pylint: disable=R0915
     config_subparsers = config_parser.add_subparsers()
 
     config_set_parser = config_subparsers.add_parser('set', description='set configuration variable')
-    config_set_parser.add_argument('key', type=str, help='variable name')
-    config_set_parser.add_argument('value', type=str, help='variable value')
+    config_set_parser.add_argument('--key', type=str, help='variable name')
```
Comment (kirillmakhonin) [inline legion/legion/cli.py]: Why did you make this params optional?

**Review by kirillmakhonin — CHANGES_REQUESTED** [review]

File: legion/legion/cli.py (hunk 2 above)
Comment (mcdoker18) [reply legion/legion/cli.py]: Fixed

File: legion/bin/legionctl (hunk 1 above)
Comment (mcdoker18) [reply legion/bin/legionctl:33]: We use such formatting in a lot of places. pylint has a warning for the case of `format`. https://docs.python.org/3/howto/logging.html#optimization

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Review by kirillmakhonin — APPROVED** [review]

**Review by alexander-semenets — APPROVED** [review]

**Pull request merged**